src/css/components/_c-post.scss

```diff
@@ -196,6 +231,11 @@ $_c-post-vertical-spacing: 4ex;
 	@include link-states($color-black);
 
 	text-underline-offset: 0.2em;
+
+	// User Queries
+	@media (prefers-color-scheme: dark) {
+		@include link-states($color-white);
+	}
```

Comment:
I think this and line 231 needs to be updated. Right now it's ending up being `var(--color-#000000)` because that's what $color-black refers to. Probably need a separate override.